code/physics/physics.cpp

```diff
@@ -150,10 +150,10 @@ void physics_set_viewer( physics_info * p, int dir )
 
 void physics_sim_rot(matrix * orient, physics_info * pi, float sim_time )
 {
-	angles	tangles;
+	angles	tangles = { 0,0,0 };
 	vec3d	new_vel;
 	matrix	tmp;
-	float		shock_amplitude;
+	float		shock_amplitude = 0.0f;
```

Comment:
Since you are adjusting these lines, anyway, might as well just move the declaration and initialization over to `line 165` to match more modern coding practices of declaring as close to the variable use as possible.